### PR TITLE
I160 Fixed typos in documentation

### LINKED ISSUE #160: Typo in documentation
## Basic Information
There's a small typo in the [writing documentation](https://elucidate-project-documentation.readthedocs.io/en/latest/#writing-documentations) section in the documentation.
'documentations' should be changed to 'documentation'.

## Other Information
FYI 'documentation' is the plural.

### PR DESCRIPTION
**Pull request by rezbyte**
## Change Summary
- Ran documentation through [LanguageTool](https://languagetool.org/), an open source alternative to Grammarly.
- Fixed spelling mistakes
- Fixed punctuation
- Fixed linting errors
- Made styling more consistent 
- Added `site_url` for canonical links in readthedocs
- Configured  indentation in markdown extension

### Change Form
- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [x] The change has documentation

## Other Information
Closes #160 

# Related issue

- Resolve #160

### CODE REVIEW

File: documentation/.markdownlint.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "MD013": false,
+  "MD033": false,
```
Comment (rezbyte) [inline documentation/.markdownlint.json:3]: `MD033` is no inline HTML by the way.

Comment (eddie-atkinson) [reply documentation/.markdownlint.json:3]: Might be worth writing a comment to that effect

**Review by eddie-atkinson — APPROVED** [review]
LGTM

**Review by jsun1590 — APPROVED** [review]
LGTM, feel free to merge

**Pull request merged**